storybook/pages/CommunityCollectibleViewPage.qml

```diff
@@ -26,12 +26,15 @@ SplitView {
                 anchors.margins: 50
                 artworkSource: ModelsData.icons.superRare
                 preview: previewBox.checked
-                deployState: mintingStateBox.checked ? 1 : 0
+                deployState: mintingStateBox.checked ? 1 /*Completed*/ : 0 /*Failed*/
+                remotelyDestructState: remotelyDestructStateBox.checked ? 1 /*In progress*/ : 2 /*Completed*/
+                burnState: burnDestructStateBox.checked ? 1 /*In progress*/ : 2 /*Completed*/
```

Comment:
I think you could use the `Constants` here too?

Reply:
Ah ok!! I will do it but in [this PR ](https://github.com/status-im/status-desktop/pull/10725) where I've changed a bit this `storybook` page! Thanks